plugins/modules/junos_interfaces.py

```diff
@@ -468,26 +811,29 @@
 #             "speed": "100m"
 #         }
 #     ]
-#
+
+
 # Using rendered
+
 - name: Render platform specific xml from task input using rendered state
   junipernetworks.junos.junos_interfaces:
     config:
     - name: ge-0/0/2
-      description: Configured by Ansibull
+      description: Configured by Ansible
       mtu: 2048
       speed: 20m
       hold_time:
         up: 3200
         down: 3200
     state: rendered
-# Task Output (redacted)
-# -----------------------
+
+# Task Output
+# -----------
 # "rendered": <nc:interfaces
 #     xmlns:nc=\"urn:ietf:params:xml:ns:netconf:base:1.0\">
 #     <nc:interface>
 #         <nc:name>ge-0/0/2</nc:name>
-#         <nc:description>Configured by Ansibull</nc:description>
+#         <nc:description>Configured by Ansible</nc:description>
```

Comment:
Let's Add documentations for rendered, gathered and parsed :
rendered:
  description: The provided configuration in the task rendered in device-native format (offline).
  returned: when I(state) is C(rendered)
  type: list
  sample:
    - <nc:protocols xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0">
gathered:
  description: Facts about the network resource gathered from the remote device as structured data.
  returned: when I(state) is C(gathered)
  type: list
  sample: >
    This output will always be in the same format as the
    module argspec.
parsed:
  description: The device native config provided in I(running_config) option parsed into structured data as per module argspec.
  returned: when I(state) is C(parsed)
  type: list
  sample: >
    This output will always be in the same format as the
    module argspec.